test/DebuggerCommon/rlexe.xml

```diff
@@ -1370,4 +1370,25 @@
       <tags>exclude_dynapogo</tags>
     </default>
   </test>
+  <test>
+    <default>
+      <files>wasm_basic.js</files>
```

Comment:
I would like it to run with jshost at one point. I just don't have the time to figure out what needs to be done right now